elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/hibernate/hql/AbstractHQLQueryBuilder.java

```diff
@@ -98,6 +101,11 @@ public AbstractHQLQueryBuilder withPossiblePagination(final Optional<Pagination>
         return this;
     }
 
+    public AbstractHQLQueryBuilder withRelationsIncludedInProjection(final Set<String> relationsIncludedInProjection) {
```

Comment:
Probably cleaner to just have withEntityProjection and remove all the withSorting, withPagination, and this new method.  Also, you can move the functionality to get the relationship names out of the EntityProjection and into this new builder function.